### PR TITLE
Build Anaconda packages 

### PR DESCRIPTION
**Pull request by jan-janssen**

### CODE REVIEW

**Comment by jan-janssen** [comment]
Maybe submitting the package to conda forge is the better idea: https://github.com/conda-forge/staged-recipes/pull/5508 

I am just wondering who should be the maintainer of the package: 
https://github.com/jan-janssen/staged-recipes/tree/box
9. What does being a conda-forge feedstock maintainer entail?
The maintainers "job" is to:
- keep the feedstock updated by merging eventual maintenance PRs from conda-forge's bots;
- keep the package updated by bumping the version whenever there is a new release;
- answer eventual question about the package on the feedstock issue tracker.

File: .travis.yml [hunk 1]
```diff
@@ -1,15 +1,43 @@
...
-    - "3.5"
-    - "3.6"
+matrix:
+  include:
+    - os: linux
+      language: python
+      env: MINICONDA="https://repo.continuum.io/miniconda/Miniconda2-latest-Linux-x86_64.sh" PYTHONVER="2.7"
+    - os: linux
+      language: python
+      env: MINICONDA="https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh" PYTHONVER="3.5"
```
Comment (cdgriffith) [inline .travis.yml]: does miniconda not support 3.3 and 3.4 or just choosing to not support older versions?

**Comment by cdgriffith** [comment]
Very good idea, I am fine with you being the conda maintainer if that is a responsibility you want. 

Only request I have is to set this PR against Development instead of master and I'll merge! (please also feel free to add yourself to the authors file, I will do that automatically if you do not as an fyi)

**Comment by jan-janssen** [comment]
@cdgriffith I am sorry for the confusion. While I first thought it was reasonable to have an own anaconda channel, so people can install python box like `conda install -c cdgriffith python-box` I learned that submitting to conda-forge is the more 'pythonic' way. Therefore I opened a pull request on conda forge and added you to the list of maintainers https://github.com/conda-forge/staged-recipes/pull/5508 . There is no need to merge this request, I guess we can just close it.

**Comment by cdgriffith** [comment]
Sorry I didn't respond earlier, thought I did but then saw this was still open, wooops.

Thank you for taking care of adding it to conda! I am assuming for every update the maintainer has to do another PR and update their code?

**Pull request closed without merging**